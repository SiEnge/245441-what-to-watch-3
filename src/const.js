export const MINUTE_IN_ONE_HOUR = 60;
export const TEN_MINUTES = 10;

export const AuthorizationStatus = {
  AUTH: `AUTH`,
  NO_AUTH: `NO_AUTH`
};

export const AppRoute = {
  LOGIN: `/login`,
  REVIEW: `/review`,
  MY_LIST: `/mylist`,
  FILMS: `/films`,
  PLAYER: `/player`,
  ROOT: `/`,
};

export const TabsType = {
  OVERVIEW: `Overview`,
  DETAIL: `Details`,
  REVIEWS: `Reviews`
};

export const ScoreType = {
  BAD: `Bad`,
  NORMAL: `Normal`,
  GOOD: `Good`,
  VERY_GOOD: `Very good`,
  AWESOME: `Awesome`,
};

// const GenreType = {
//   ALL_GENRES: `All genres`,
//   COMEDIAS: `Comedies`,
//   CRIME: `Crime`,
//   DOCUMENTARY: `Documentary`,
//   DRAMAS: `Dramas`,
//   HORROR: `Horror`,
//   ROMANCE: `Romance`,
//   SCI_FI: `Sci-Fi`,
//   THRILLERS: `Thrillers`,
// };
