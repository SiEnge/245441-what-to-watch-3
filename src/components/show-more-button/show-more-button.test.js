import React from "react";
import renderer from "react-test-renderer";
import ShowMoreButton from "./show-more-button.jsx";
import {noop} from "../../const.js";

it(`Render ShowMoreButton`, () => {
  const tree = renderer
    .create(<ShowMoreButton
      onShowMoreButtonClick={noop}
    />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});

