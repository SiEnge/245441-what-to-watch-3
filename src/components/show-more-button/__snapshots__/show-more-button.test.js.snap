// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render ShowMoreButton 1`] = `
<div
  className="catalog__more"
>
  <button
    className="catalog__button"
    onClick={[Function]}
    type="button"
  >
    Show more
  </button>
</div>
`;
